README.md

```diff
@@ -67,10 +56,8 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+To Microverse and their support team.
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] **REQUIRED:** Please kindly personalize your licence by replacing your name, app name and removing unnecessary lines, because it is not good for a professional project.